packages/manager/apps/pci-kubernetes/src/pages/detail/nodepools/new/New.page.tsx

```diff
@@ -260,7 +260,7 @@ export default function NewPage(): JSX.Element {
       <StepComponent
         id={StepsEnum.NAME}
         order={1}
-        title={t('add:kubernetes_add_name_title')}
+        title={t('add:kube_add_node_pool_name_title')}
```

Comment:
We should update associated test here: https://github.com/ovh/manager/blob/2493c68139675291f341fe7e4386cebefb0db5c8/packages/manager/apps/pci-kubernetes/src/pages/new/New.page.spec.tsx#L126